pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -931,7 +936,10 @@ func ValidateEndTimeAndTruncate(
 				)
 			}
 		}
-		return defaultURIs[:i+1], mainBackupManifests[:i+1], localityInfo[:i+1], nil
+		if !includeSkipped {
+			return defaultURIs[:i+1], mainBackupManifests[:i+1], localityInfo[:i+1], nil
+		}
+		return ElideSkippedLayers(defaultURIs[:i+1], mainBackupManifests[:i+1], localityInfo[:i+1])
```

Comment:
Isn't this flipped? Should the condition be `if includeSkipped`?

Reply:
Yep.